content/applications/finance/fiscal_localizations/france.rst

```diff
@@ -337,6 +371,108 @@ specific to France:
+accounting standards and is usually done with the help of accountants or tax professionals.
+
+.. _localization/france/teledec:
+
+Teledec
+-------
+
+Teledec is a tool to prepare and submit tax returns by utilizing data from accounting records. To
+electronically send the company's liasse fiscale to the DGFiP (Direction Générale des Finances
+Publiques) and synchronize the information between Teledec and Odoo, follow these steps:
+
+#. :ref:`localization/france/teledec-account`
+#. :ref:`localization/france/teledec-registration`
+#. :ref:`localization/france/teledec-synchronization`
+
+.. _localization/france/teledec-account:
+
+Teledec account creation
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To create a Teledec account, open `Teledec <https://www.teledec.fr/s-enregistrer>`_ and fill in the
+:guilabel:`Adresse e-mail` field with an email address. Choose a secure password, accept the general
+terms and conditions by checking the box, and click :guilabel:`S'enregistrer` to save. Then, enter
+the SIREN number of the company.
+
+.. note::
+   If the account has already been created, click :guilabel:`Se connecter` (Connect) at the top of
+   the page.
+
+.. _localization/france/teledec-registration:
+
+Company registration and fiscal year information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To register the company in Teledec, go to :guilabel:`Vos entreprises` (Your companies) and click
+:guilabel:`Enregistrer votre entreprise` (Register your company). Make sure to fill in the following
+company information in the :guilabel:`Coordonnées de l'entreprise` (Company's details) and
+:guilabel:`Représentant légal` (Legal representative) sections:
+
+- :guilabel:`Nom de l'entreprise`: company's name
+- :guilabel:`Forme juridique`: company's legal form to select
+- :guilabel:`Les comptes sont clôturés le`: closing date
+- :guilabel:`Régime d'imposition, choix de la liasse`: Tax scheme and tax return choice to select
+- :guilabel:`Adresse du siège social`: Head office address
+- :guilabel:`Nom du représentant légal`: Legal representative's name
+- :guilabel:`Agissant en qualité de`: legal representative's function
+- :guilabel:`Numéro de téléphone`: phone number
+
+Click :guilabel:`Sauvegarder` (Save) to open the following
+:guilabel:`Enregistrer les informations sur l'exercice` step. Then, fill in information on the
+financial year such as the fiscal year start and end dates or the closing date and duration of the
+previous fiscal period.
+After saving, the list of documents included in the liasse fiscale will be accessible, including
+both standard tax forms and those customized for the company’s tax return.
+
+.. note::
+   - The :guilabel:`Etat` (Status) column shows the progress of the document filing.
+   - To print a blank version of the declaration, click :guilabel:`Imprimer la déclaration` and
+     select the :guilabel:`Imprimer la déclaration avec les notices` option.
+
```

Comment:
I would use a tip instead of a note and add a list item about how you can click "Compléter" to fill out a document (and then need to save). What do you think?